ci/build_common.sh

```diff
@@ -23,11 +23,44 @@ readonly GPU_ARCHS=$(echo $3 | tr ' ,' ';')
 
 readonly PARALLEL_LEVEL=${PARALLEL_LEVEL:=$(nproc)}
 
+BUILD_DIR=${BUILD_DIR:-../build/${DEVCONTAINER_NAME:-}}
```

Comment:
If no DEVCONTAINER_NAME I would go with `local` or `unknown` instead of nothing. That way all builds are in subdirectories of `build/` and you don't have this edge case

Reply:
I've updated it so building outside a container will build to `cccl/build/local`. Furthermore, I've also made it so the most recent build will always be symlinked to `cccl/build/latest` so there is a consistent entry point to the most recent build.